package.json

```diff
@@ -29,18 +29,20 @@
     "postdoc": "mkdocs build",
     ":clean:doc": "git clean -fdX ./site ./docs",
     "autolint": "eslint .",
-    "lint": "cross-env STRICT_LINT=1 npm run autolint --",
-    "prebuild": "npm run-script autolint",
+    "lint": "npm run lint:js && npm run lint:css",
+    "lint:js": "cross-env STRICT_LINT=1 npm run autolint --",
+    "lint:css": "stylelint '{docs,src}/**/*.css'",
```

Comment:
This should probably go under `autolint`, but we'd have to figure out a way to run `stylelint` in warn-only mode like we do for `eslint`. I'm also a little worried about that glob, since I'm not sure it'd work on Windows...